src/structures/node.jl

```diff
@@ -96,43 +154,45 @@ GenAvailability(id, 𝒫::Vector{<:Resource}) = GenAvailability(id, 𝒫, 𝒫)
 """ A reference `Storage` node.
 
 This node is designed to store either a `ResourceCarrier` or a `ResourceEmit`.
-It is designed as a composite type to automatically distinguish between these two.
+It is designed as a parametric type to differentiate between different cyclic behaviours.
```

Comment:
This is encoded with the type parameter `T`, right? Maybe mention that explicitly.

Reply:
I agree, I extended the docstring.